src/runtime/basic-runtime.ts

```diff
@@ -243,6 +244,7 @@ export class BasicRuntime implements BasicSubscriptions {
       autoPromptType,
       alwaysShow,
       isClosable,
```

Comment:
@kristenwang Are we ok updating this value for OnsitePreview to also use `contentType`? I believe in today's flow we use `isClosable`, but this will be deprecated.

Reply:
Note: we can no longer assume the contribution prompt will be present in the audience actions for all contribution revenue models.